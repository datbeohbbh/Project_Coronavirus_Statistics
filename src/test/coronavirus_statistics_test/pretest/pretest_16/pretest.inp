28 25
q 1 14 380
q 1 7 392
q 3 14 326
q 1 28 204
q 2 14 31
q 1 3 447
? 4 3
q 3 7 184
? 4 2
q 2 7 112
q 3 3 356
q 2 3 44
q 2 28 59
q 1 22 386
? 7
q 1 28 297
q 2 22 119
? 4 4
? 4 5
? 4 5
? 4 5
? 6
? 6
? 4 5
? 7