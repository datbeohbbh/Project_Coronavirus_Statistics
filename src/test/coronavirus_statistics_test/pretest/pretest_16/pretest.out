Query ? 4 3 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 392
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 204
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 3
Quantity: 356 people

Query ? 4 4 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 267
-> Has recovered : 119
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 442
-> Has recovered : 59
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 442
-> Has recovered : 59
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 442
-> Has recovered : 59
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 22
Quantity: 119 people

Query ? 6 :
The city with the highest recovery is city with the index 22
Quantity: 119 people

Query ? 4 5 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 442
-> Has recovered : 59
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 3
Quantity: 356 people

